All The Flavors/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "Nekolen",
+        "id": "270125856679002113"
+    },
+    "service": "alltheflavors",
```

Comment:
Name the service the same as the folder name

Reply:
I did it at first but if i put space in the service, it doesn't load in presence launcher

Reply:
![image](https://user-images.githubusercontent.com/34792838/66408364-1b8a1300-e9ef-11e9-8e89-3dcf5af0b618.png)
https://premid.app/store/presences/Google%20Docs

Works on google docs

Reply:
sure then